packages/x-tree-view-pro/package.json

```diff
@@ -47,8 +47,7 @@
     "@mui/base": "^5.0.0-beta.40",
     "@mui/system": "^5.15.14",
     "@mui/utils": "^5.15.14",
-    "@mui/x-license": "7.0.0",
-    "@mui/x-tree-view-pro": "7.0.1",
```

Comment:
I probably removed it prematurely. 🙈 
Should it have been `@mui/x-tree-view` here instead as we are re-exporting its contents? 🤔 
cc @flaviendelangle

Reply:
Yes indeed, my bad
No problem to update the version here, but the package should not be published since it's `private: true`